addons/istio/istio.yaml

```diff
@@ -26,7 +26,7 @@ spec:
     - matchLabels:
         kubeaddons.mesosphere.io/name: prometheus
   kubernetes:
-    minSupportedVersion: v1.16.0
+    minSupportedVersion: v1.17.0
```

Comment:
so we can only deploy this addon on konvoy 1.5.x and newer. But I guess since its preview it should be okay.

Reply:
why we have to bump the `minSupportedVersion`?

Reply:
I think I did this errantly, fixing
```suggestion
    minSupportedVersion: v1.16.0
```